src/main/java/io/crate/jmx/http/HttpServer.java

```diff
@@ -119,6 +121,7 @@ public void start(boolean daemon) {
     /**
      * Stop the HTTP server.
      */
+// TODO: Should we add some shutdown hook for graceful shutdown?
```

Comment:
afaik stopping is not needed as the agent is killed when the JVM stops... afaik we could remove this method at all